sdk/tvOS/OoyalaTVSkinSDK/OOTVGestureManager.m

```diff
@@ -89,4 +116,32 @@ - (void)togglePlay:(id)sender {
   [self.controller showProgressBar];
 }
 
+- (void)onSwipe:(id)sender {
+  if (sender == self.panGesture) {
+    CGPoint currentPoint = [self.panGesture translationInView:self.controller.view];
+    CGFloat seekScale = self.controller.player.duration / 1920.0;
```

Comment:
yes, you are right, I should use view width. Actually we should allow users to configure the multiplier.